features/users/user_rename.feature

```diff
@@ -210,4 +227,4 @@ Feature:
         And I fill in "Password" with "secret"
         And I press "Change User Name"
       Then I should get confirmation that I changed my username
-        And I should see "Hi, notforbidden"
+        And I should see "Hi, notforbidden"
```

Comment:
Would be nice if you could add the newline at the end of the file back in.

Reply:
Done!